### DISCUSSION TITLE
no notification of missed calls on home screen

### DISCUSSION THREAD

**Discussion by jimbox51** [FossifyOrg/General-Discussion#530] [General]
Have I missed something? I am not getting the "dot" notification for missed calls. I have "Allow notification dot" set for this app.
but it I never seem to get a "dot". Aother apps get dots for notifications.

I'm on Android 14 on a Fairphone 5 and am using the version of the app from F-Droid.

**Comment by Aga-C** [comment]
Currently, it's missing, but we already have a feature request for it: https://github.com/FossifyOrg/Phone/issues/83.
